function.rb

```diff
@@ -0,0 +1,9 @@
+
+def function(n)
+  if n == 0
```

Comment:
if n is a float, this function might go on forever since it will never == zero. Perhaps <= would be better?